### PR TITLE
feat: add new threat shield configurations

### PR DESCRIPTION
**Pull request by andre8244**
Ref:
- https://github.com/NethServer/nethsecurity/issues/795

### CODE REVIEW

**Comment by andre8244** [comment]
Let's fix *Lorem ipsum*s before merging

File: public/i18n/en/translation.json
Comment (gsanchietti) [inline public/i18n/en/translation.json:1485]: Brute force attack protection safeguards against repeated unauthorized login attempts by blocking or limiting access after multiple failed attempts.

File: public/i18n/en/translation.json
Comment (gsanchietti) [inline public/i18n/en/translation.json:1486]:
```suggestion
      "block_brute_force_tooltip": "When enabled, the system will scan logs for patterns of failed access attempts.",
```

File: public/i18n/en/translation.json
Comment (gsanchietti) [inline public/i18n/en/translation.json:1488]:
```suggestion
      "ban_after_n_attempts_tooltip": "The number of times an IP must appear in the log before being flagged as suspicious and blocked.",
```

File: public/i18n/en/translation.json
Comment (gsanchietti) [inline public/i18n/en/translation.json:1490]:
```suggestion
      "attack_patterns_tooltip": "Enter a regular expression to search the logs; if matched, the source IP will be marked as suspicious.",
```

File: public/i18n/en/translation.json
Comment (gsanchietti) [inline public/i18n/en/translation.json:1493]:
```suggestion
      "block_dos_description": "DoS (Denial of Serevice) limits excessive suspicious requests from a single source",
```

File: public/i18n/en/translation.json
Comment (gsanchietti) [inline public/i18n/en/translation.json:1501]:
```suggestion
      "ban_time_tooltip": "After the timeout period, the blocked IP will be automatically unblocked, restoring its access to the system.",
```

File: src/components/standalone/dashboard/ThreatShieldIpCard.vue
Comment (Tbaile) [inline src/components/standalone/dashboard/ThreatShieldIpCard.vue:204]: I'd use a RouterLink component here (naturally wrapped by a NeLink for styling)
https://router.vuejs.org/guide/#App-vue

File: src/views/standalone/security/ThreatShieldView.vue
Comment (Tbaile) [inline src/views/standalone/security/ThreatShieldView.vue:18]: Maybe `BanIpLocalAddress` is better?

It's your call on this one

Comment (andre8244) [reply src/views/standalone/security/ThreatShieldView.vue]: Done

**Review by Tbaile — APPROVED** [review]

**Pull request merged**
